### ISSUE TITLE
Building MinGW64-Qt5: [Makefile:130: all] Error 2

### ISSUE THREAD

**Issue by SantosSi** [manisandro/gImageReader#355]
After 5a0105c84c43158bcbd4db7e841561903f3ed6ef the above error appears at 56% in the Docker build environment.

**Comment by manisandro** [comment]
Please post the full error (or the full build log if you can't locate the error).

**Comment by SantosSi** [comment]
Is this enough:
>>
usr/bin/msgfmt -o /workspace/build/mingw64-qt5/zh_CN.gmo /workspace/build/mingw64-qt5/zh_CN.po
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/zh_HK.gmo /workspace/build/mingw64-qt5/zh_HK.po
[ 23%] Generating zh_TW.gmo
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/zh_TW.gmo /workspace/build/mingw64-qt5/zh_TW.po
make -f CMakeFiles/gimagereader.dir/build.make CMakeFiles/gimagereader.dir/depend
make[2]: Entering directory '/workspace/build/mingw64-qt5'
[ 24%] Generating pt.gmo
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/pt.gmo /workspace/build/mingw64-qt5/pt.po
[ 25%] Generating qrc_gimagereader.cpp
/usr/x86_64-w64-mingw32/bin/qt5/rcc --name gimagereader --output /workspace/build/mingw64-qt5/qrc_gimagereader.cpp /workspace/qt/data/gimagereader.qrc
RCC: Error in '/workspace/qt/data/gimagereader.qrc': Cannot find file '../../data/icons/angle.png'
make[2]: *** [CMakeFiles/gimagereader.dir/build.make:126: qrc_gimagereader.cpp] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 26%] Generating ui_AboutDialog.h
/usr/x86_64-w64-mingw32/bin/qt5/uic -o /workspace/build/mingw64-qt5/ui_AboutDialog.h /workspace/qt/data/AboutDialog.ui
[ 27%] Generating ko.gmo
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/ko.gmo /workspace/build/mingw64-qt5/ko.po
[ 28%] Generating sr_Cyrl.gmo
/usr/bin/cmake -P /workspace/cmake/gettextizeui.cmake /workspace/build/mingw64-qt5/ui_AboutDialog.h
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/sr_Cyrl.gmo /workspace/build/mingw64-qt5/sr_Cyrl.po
-- Gettextizing /workspace/build/mingw64-qt5/ui_AboutDialog.h
make[2]: Leaving directory '/workspace/build/mingw64-qt5'
make[1]: *** [CMakeFiles/Makefile2:173: CMakeFiles/gimagereader.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 29%] Generating sr_CS.gmo
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/sr_CS.gmo /workspace/build/mingw64-qt5/sr_CS.po
[ 30%] Generating sr_Latn.gmo
/usr/bin/msgfmt -o /workspace/build/mingw64-qt5/sr_Latn.gmo /workspace/build/mingw64-qt5/sr_Latn.po
make[2]: Leaving directory '/workspace/build/mingw64-qt5'
Merging translations into /workspace/build/mingw64-qt5/data/gimagereader-qt5.desktop.
make[2]: Leaving directory '/workspace/build/mingw64-qt5'
[ 57%] Built target potfiles_1
[ 57%] Built target desktop_file
make[1]: Leaving directory '/workspace/build/mingw64-qt5'
make: *** [Makefile:130: all] Error 2
<<

**Comment by manisandro** [comment]
Yes - argh looks like one of those mysterious docker filesystem oddities. The file it is complaining about is definitely there.

**Comment by manisandro** [comment]
Looks like possibly the resolution of relative paths in mounted volumes is being prevent by some security mechanism in Docker? In any event, this works:

    docker run --security-opt seccomp:unconfined -v $PWD:/workspace gimagereader_buildenv packaging/win32/makeinstaller.sh x86_64 qt5

i.e. with the added `--security-opt seccomp:unconfined`. I need to investigate further whether there is a better way.

**Comment by SantosSi** [comment]
It happens even after deleting the containers and the image and then recreating it.
